src/platforms/android/configuration/app-not-respond.mdx

```diff
@@ -4,6 +4,21 @@ sidebar_order: 3
 description: "Learn how to turn off or specify ANR."
 ---
 
+The Android SDK uses two implementations to detect ANRs:
+
+- Reading [ApplicationExitInfo](https://developer.android.com/reference/android/app/ApplicationExitInfo) for devices running Android 11 and above (v2)
+- A Watchdog thread for devices running Android version below 11 (v1)
+
+The new implementation (v2) uses the same data source as Google Play Console, so the reported ANR events and [ANR rate](/platforms/android/performance/instrumentation/performance-metrics/#application-not-responding-anr-rate) should match with what you see on Sentry. We also can capture a thread dump with some additional information like held locks to help you resolve ANRs more efficiently.
+
+<Note>
+
+The v1 approach (Watchdog) still has some advantages over v2, for example, capturing screenshots and transactions with profiles at the time of ANR. Therefore, we're considering support both approaches working alongside each other. Please, upvote this [discussion](https://github.com/getsentry/sentry-java/discussions/2716) and share your feedback, if you have a case for v1 and v2 working together.
+
+</Note>
```

Comment:
```suggestion
The original Watchdog approach (v1) has some advantages over v2, like capturing screenshots and transactions with profiles at the time of ANR.

<Note>

We're considering SDK support for both approaches working alongside each other on Android 11 and up. Please upvote [this GitHub discussion](https://github.com/getsentry/sentry-java/discussions/2716) and share your feedback if you have a case for v1 and v2 working together.

</Note>
```

Reply:
What do you think about splitting only this into the note? I feel like it makes more sense this way.